pkg/controller/acicontainersoperator.go

```diff
@@ -379,10 +432,77 @@ func (c *Controller) handledaemonsetUpdate(oldobj interface{}, newobj interface{
 			queue.Add(key)
 		}
 	} else {
-		log.Info("Owner Reference is intact for ", new_ds.Name)
+		log.Info("Owner Reference is unchanged for ", new_ds.Name)
 	}
 }
 
+func (c *Controller) handleConfigMapCreate(newobj interface{}) bool {
+	new_config := newobj.(*corev1.ConfigMap)
+	if _, err := os.Stat("/tmp/aci-operator.conf"); os.IsNotExist(err) {
+		log.Info("File not present. Writing initial aci-operator-config configmap")
+		err := c.WriteConfigMap("/tmp/aci-operator.conf", new_config)
+		if err != nil {
+			return true
+		}
+		return false
+	}
+
+	log.Info("Writing new aci-operator-config configmap")
+	err := c.WriteConfigMap("/tmp/aci-operator.conf", new_config)
+	if err != nil {
+		return true
+	}
+
+	log.Info("Reading current aci-operator-config configmap")
+	rawSpec, err := c.ReadConfigMap("/tmp/aci-operator.conf")
+	if err != nil {
+		return true
+	}
+
+	obj := c.CreateAciContainersOperatorObj()
+	log.Info("Unmarshalling the ConfigMaps...")
```

Comment:
name of configmap...also notice how sometimes you use "Config-map" and then sometimes you say "ConfigMaps"...admittedly a nit, but there has to be consistency in conventions at least in the same module...